ncclient/manager.py

```diff
@@ -167,12 +167,12 @@ def wrapper(self, *args, **kwds):
             wrapper.__doc__ = op_cls.request.__doc__
             return wrapper
         if VENDOR_OPERATIONS:
-            for op_name, op_cls in VENDOR_OPERATIONS.items():
+            for op_name, op_cls in six.iteritems(VENDOR_OPERATIONS):
                 setattr(cls, op_name, make_wrapper(op_cls))
         return super(OpExecutor, cls).__call__(*args, **kwargs)
 
 
-class Manager(object, metaclass=OpExecutor):
+class Manager(six.with_metaclass(OpExecutor,object )):
```

Comment:
please change with_metaclass(OpExecutor,object ) to
with_metaclass(OpExecutor, object)  as per pep8 standard